subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/DependencyReportTaskIntegrationTest.groovy

```diff
@@ -848,6 +849,55 @@ api (n)
 \\--- foo:foo:1.0 (n)
 
 (n) - Not resolved (configuration is not meant to be resolved)
+"""
+    }
+
+    @NotYetImplemented
```

Comment:
:o: I would prefer to have this ` @ToBeImplemented` with an adjusted assertion. To not forget about this test. E.g. if we implement this with slightly different formatting, this test will still fail without us noticing.

Reply:
I have not been personally convinced that `@ToBeImplemented` is any better: https://gradle.slack.com/archives/C13AA53CK/p1505937273000344